scripts/ping360_socat.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python -u
```

Comment:
Since we are moving to python3 and python2 is deprecated, can we use this script in python3 ?

Reply:
I think we should have a PR to move everything. Python3 doesn't have the ping library installed yet. I'd prefer moving everything at once and not having two copies of the library, one for Python2 and one for Python3.

Reply:
this file does not use such libraries.